package.json

```diff
@@ -49,42 +51,42 @@
     ]
   },
   "devDependencies": {
-    "@types/chai": "3.4.30",
-    "@types/enzyme": "2.4.30",
-    "@types/mocha": "2.2.29",
-    "@types/react-addons-test-utils": "0.14.14",
-    "@types/sinon": "1.16.26",
-    "awesome-typescript-loader": "2.2.3",
+    "@types/chai": "3.4.34",
```

Comment:
According to the docs, that's not true:

> Our package exposes declarations from each of those, so any user of our browserify-typescript-extension package needs to have these dependencies as well. For that reason, we used "dependencies" and not "devDependencies", because otherwise our consumers would have needed to manually install those packages. If we had just written a command line application and not expected our package to be used as a library, we might have used devDependencies.

https://www.typescriptlang.org/docs/handbook/declaration-files/publishing.html#dependencies